File: doc-examples/jdbc-example-java/src/main/java/example/StudentClass.java
package example;

import io.micronaut.data.annotation.GeneratedValue;
import io.micronaut.data.annotation.Id;
import io.micronaut.data.annotation.MappedEntity;

@MappedEntity(value = "TBL_STUDENT_CLASSES", alias = "sc")
public class StudentClass {
    @Id
    @GeneratedValue(GeneratedValue.Type.IDENTITY)
    private Long id;
}

File: doc-examples/jdbc-example-python/src/main/python/example/StudentClass.py
from dataclasses import dataclass
from typing import Annotated

from micronaut.data.annotation import GeneratedValue, Id, MappedEntity


@MappedEntity(value="TBL_STUDENT_CLASSES", alias="sc")
@dataclass
class StudentClass:
    id: Annotated[int | None, Id, GeneratedValue("IDENTITY")] = None
